src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-delete.ts

```diff
@@ -1,20 +1,25 @@
-import { NextFunction, Request, Response } from "express";
+import { Request, Response } from "express";
 import { GitHubServerApp } from "~/src/models/github-server-app";
 
 export const JiraConnectEnterpriseAppDelete = async (
 	req: Request,
-	res: Response,
-	next: NextFunction
+	res: Response
 ): Promise<void> => {
 	try {
 		req.log.debug("Received Jira Connect Enterprise App DELETE request");
 
-		await GitHubServerApp.uninstallApp(req.body.uuid);
+		const { gitHubAppConfig } = res.locals;
+		if (!gitHubAppConfig || !gitHubAppConfig.uuid) {
```

Comment:
Just double check the uuid is valid and not cloud

Reply:
Shouldn't it be `.json()` ?

Reply:
Also, would it make sense to add some logging for all the error cases for easier investigation?

Reply:
fixed.